src/gui/plugins/scene_manager/GzSceneManager.cc

```diff
@@ -132,6 +132,12 @@ void GzSceneManager::Update(const UpdateInfo &_info,
     this->dataPtr->renderEnginePluginPathsInit = true;
   }
 
+  if (!this->dataPtr->renderEnginePluginPathsInit)
+  {
+    this->dataPtr->renderUtil.InitRenderEnginePluginPaths();
+    this->dataPtr->renderEnginePluginPathsInit = true;
+  }
```

Comment:
this block is a duplicate of the one right above. Is this intentional?

Reply:
[ffe2da0](https://github.com/gazebosim/gz-sim/pull/1794/commits/ffe2da087fb41925a0fab13ee063774c1c93cae0)